tests/ui/feature-gates/feature-gate-raw-dylib-elf.rs

```diff
@@ -0,0 +1,8 @@
+//@ only-elf
```

Comment:
The `x86_64-fortanix-unknown-sgx` target doesn't support dynamic linking. Can you please add `//@ ignore-sgx` to ignore this platform?